src/lib/ops/policyOps.js

```diff
@@ -145,18 +145,35 @@ const policyOps = {
    * @param  {Function} cb
    */
   listByOrganization: function listByOrganization (params, cb) {
-    const { organizationId } = params
+    const { organizationId, limit, page } = params
 
```

Comment:
The protection on incorrect data is from Joi validation.
From what I saw on internet 0 is not a positive number but the `integer().positive()` validation from Joi from what i saw in code has something like `Hoek.assert(Hoek.isInteger(limit) && limit >= 0, 'limit must be a positive integer');`

I think there are chances to receive 0 values. We should add more protection on Joi and also apply it on my implementation.